app/src/main/res/layout/multiple_choice_interaction_items.xml

```diff
@@ -39,6 +39,7 @@
       android:layout_toEndOf="@+id/multiple_choice_radio_button"
```

Comment:
we can reformat this file too.

Reply:
Yes, I avoided this earlier just to keep PR review easy. Will now make a commit to fix all reformatting.